web/src/timerender.ts

```diff
@@ -445,26 +438,6 @@ export function stringify_time(time: number | Date): string {
     return get_localized_date_or_time_for_format(time, "time");
 }
 
-export function format_time_modern(time: number | Date, today = new Date()): string {
```

Comment:
Let's not remove this; as noted in 7bc0e70693f49ea8faef2578158362491fe5470c, we probably do want to use this function somewhere -- we're just a little stuck on product decisions for how we want to use it.

Reply:
Dropped the commit